create-cloudflare/next/package.json

```
@@ -0,0 +1,33 @@
+{
+	"name": "next",
+	"version": "0.1.0",
+	"private": true,
+	"scripts": {
+		"dev": "next dev --turbopack",
+		"build": "next build",
+		"start": "next start",
+		"lint": "next lint",
+		"deploy": "opennextjs-cloudflare build && opennextjs-cloudflare deploy",
+		"upload": "opennextjs-cloudflare build && opennextjs-cloudflare upload",
+		"preview": "opennextjs-cloudflare build && opennextjs-cloudflare preview",
+		"cf-typegen": "wrangler types --env-interface CloudflareEnv ./cloudflare-env.d.ts"
+	},
+	"dependencies": {
+		"@opennextjs/cloudflare": "^1.11.0",
+		"next": "15.4.6",
```

The c3 template was not updated because of a CI error on Linux.
Turns out that was only because turbopack is used by default on Linux.
This should be fixed by https://github.com/cloudflare/workers-sdk/pull/11014 and we can update here
_Note that the e2e tests in this repo are already running [15.5.6](https://github.com/opennextjs/opennextjs-cloudflare/blob/73b711187b2dffb67db4dfeec05df2fbc6506bbb/pnpm-workspace.yaml#L48)_  

```suggestion
		"next": "15.5.6",
```